### ISSUE TITLE
Switch capistranorb.com to https

### ISSUE THREAD

**Issue by will-in-wi** [capistrano/capistrano#1990]
GitHub Pages supports it now: https://blog.github.com/2018-05-01-github-pages-custom-domains-https/

Looks like we are using A records (probably because we are using the root domain instead of a subdomain like www). So I'll update dnssimple with the new A records, enable https, verify it works, make any fixes as needed, then redirect to https by default.

Any objections? cc/ @mattbrictson @leehambley

**Comment by mattbrictson** [comment]
Absolutely 👍 Thanks for looking into this!

**Comment by leehambley** [comment]
🤘👍👍

**Comment by will-in-wi** [comment]
DNS is cut over. I switched the apex domain over to the new A records, and I switched the `www` redirect subdomain over to a CNAME from the previous A records. It looks like GitHub has issues a Let's Encrypt cert at this point.

I'm going to give this some time for the DNS changes to propagate, then audit the site for any `http` dependencies that'll cause mixed content warnings. Finding none (or fixing the ones that exist), I'll enforce https.
